src/app/Components/MainDashbordComponents/footer/footer.component.css

```diff
@@ -38,24 +38,24 @@ footer {
 }
 
 .btn1:hover {
```

Comment:
primary-button-hover class should be in common css and called that class everywhere